EventFilter/TotemRawToDigi/python/TotemRPRawToDigi_cfi.py

```diff
@@ -0,0 +1,6 @@
+import FWCore.ParameterSet.Config as cms
+
+from EventFilter.TotemRawToDigi.TotemVFATRawToDigi_cfi import totemVFATRawToDigi
+
+totemRPRawToDigi = totemVFATRawToDigi.copy()
+totemRPRawToDigi.subSystem = "RP"
```

Comment:
this is technically a cff file, since it has more complex content including multiple module instances and copies